server/cluster/metadata/table_manager_test.go

```diff
@@ -1,7 +1,7 @@
 // Copyright 2022 CeresDB Project Authors. Licensed under Apache-2.0.
 
 // nolint
-package metadata
+package metadata_test
```

Comment:
Is there this style in the Golang?

Reply:
Yes, this is a little trick for writing unit tests in Golang. Test files that declare a package with the suffix "_test" will be compiled as a separate package, and then linked and run with the main test binary.
This has two advantages. First, different package names cannot directly refer to internal variables and functions, which enables black-box testing. Second, it avoids circular dependencies.
More information can be found in this issue: https://github.com/golang/go/issues/25223

Reply:
Thanks! Learned a lot from it. However, how to test the private methods if so?

Reply:
I do not want to test private method, I think unit testing only needs to ensure the correctness of the exposed functions. If the private function is huge or complex or important enough to require its own tests, just put it in another struct and make it public there.

Reply:
I guess visibility should not be designed for tests, but let's leave the decision in the future and move on for now.